papermill/tests/test_inspect.py

```diff
@@ -0,0 +1,104 @@
+from pathlib import Path
+from unittest.mock import MagicMock, patch
+
+import pytest
+from click import Context
+
+from papermill.inspection import display_notebook_help, inspect_notebook
+
+
+NOTEBOOKS_PATH = Path(__file__).parent / "notebooks"
+
+
+def _get_fullpath(name):
+    return NOTEBOOKS_PATH / name
+
+
+@pytest.fixture
+def click_context():
+    mock = MagicMock(spec=Context, command=MagicMock())
+    mock.command.get_usage.return_value = "Dummy usage"
+    return mock
+
+
+@pytest.mark.parametrize(
```

Comment:
Probably want more examples (maybe by supplying the parameter cell directly to the inspect call in a different test parameterization) with a variety of code arrangements to prove your comment compressing code all works as expected.

Reply:
This is actually done in  papermill/tests/test_translators.py:test_inspect_python (at line 112)